app/models/data/local_authority_user.rb

```diff
@@ -2,30 +2,34 @@ module Data
   class LocalAuthorityUser
     include ToCsv
 
-    def self.to_csv
-      new.generate_csv
-    end
+    class << self
+      # @return [String]
+      def column_names
+        ['Local Authority', 'Users']
+      end
 
-    # @return [String]
-    def generate_csv
-      CSV.generate do |csv|
-        csv << ['Local Authority', 'Users']
-        count_by_local_authority.each do |local_authority, count|
-          csv << [local_authority, count]
+      # @return [Array<Hash{Symbol => Mixed}>]
+      def dashboard
+        data = []
+        count_by_local_authority.each do |authority, count|
```

Comment:
Same again for using `map` but also we should add `private`

Reply:
[0c368b6](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/648/commits/0c368b6eb789a1cf6247a6bd7d09d4d90a2cfd9b)